File: android/src/main/java/com/cactuslab/capacitor/nativenavigation/types/PopResult.java
package com.cactuslab.capacitor.nativenavigation.types;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

public class PopResult {
    @NonNull
    public String stack;
    @Nullable
    public String id;

    public PopResult(@NonNull String stack, @Nullable String id) {
        this.stack = stack;
        this.id = id;
    }

}

File: android/src/main/java/com/cactuslab/capacitor/nativenavigation/types/PopResult.kt
package com.cactuslab.capacitor.nativenavigation.types

import com.cactuslab.capacitor.nativenavigation.types.ComponentOptions.TabOptions
import com.cactuslab.capacitor.nativenavigation.types.ModalPresentationStyle
import com.cactuslab.capacitor.nativenavigation.types.ComponentType
import com.cactuslab.capacitor.nativenavigation.types.ComponentOptions
import com.cactuslab.capacitor.nativenavigation.types.TabsOptions
import com.cactuslab.capacitor.nativenavigation.types.ViewOptions
import com.cactuslab.capacitor.nativenavigation.exceptions.MissingParameterException
import com.cactuslab.capacitor.nativenavigation.types.CreateOptions

class PopResult(var stack: String, var id: String?)